### ISSUE TITLE
postgresql Dockerfile should --follow-symlinks

### ISSUE THREAD

**Issue by JPMenezes** [georchestra/georchestra#1676]
The Dockerfile in the postgresql folder needs a --follow-symlinks on the last line as an argument to patch or else it won't patch the docker-entrypoint.sh file because it's a symlink.

Also, in the same file, there's a typo:
"environement" -> "environment"

Thank you.

**Comment by fvanderbiest** [comment]
Right, and applying your fix is no enough (at least on 15.12) since the upstream `docker-entrypoint.sh` file has changed:
```
Step 9 : RUN cd / && patch -p 0 --follow-symlinks < docker-entrypoint.sh.patch
 ---> Running in c48b68d5bca7
patching file docker-entrypoint.sh
Hunk #1 FAILED at 79.
1 out of 1 hunk FAILED -- saving rejects to file docker-entrypoint.sh.rej
The command '/bin/sh -c cd / && patch -p 0 --follow-symlinks < docker-entrypoint.sh.patch' returned a non-zero code: 1
```

**Comment by fvanderbiest** [comment]
I don't like having to patch the entrypoint ... this is too fragile.

I propose that we maintain two docker images for the purpose, one `georchestra/postgres` with test data, and another one tagged `georchestra/postgres:nodata` with no data. Opinions ?

**Comment by Vampouille** [comment]
We can also test with a shell script that will truncate (`echo "" > 01-ldap-test-data.sql`) sql files with test data if env var is not set.

**Comment by fvanderbiest** [comment]
Sounds good, indeed ! Thanks :-)

**Comment by fvanderbiest** [comment]
Fixed with 8ebf4031acc4ae1521b827f302308f8ca47b01fb
